### ISSUE TITLE
Document CONAN_EXPORTED

### ISSUE THREAD

**Issue by lasote** [conan-io/docs#518]
set by the CMake helper, what is the use case?

https://github.com/conan-io/conan/issues/2408

**Comment by ovidiub13** [comment]
The changelog when this was added says:
```
Feature: new cmake CONAN_EXPORTED variable, can be used in CMakeLists.txt to differentiate building in the local conan cache as package and building in user space
```

So I expect this to mean that we can use this variable to differentiate between building with `conan create` and `conan build`.

But I understood from @memsharded that the difference is between `conan create` and calling cmake directly.

**Comment by lasote** [comment]
Definitely confusing.

**Comment by memsharded** [comment]
Yes, agree that was very confusing. 

The use case is to differentiate when cmake is being called by conan CMake helper (e.g. assuming we have a conanbuildinfo.cmake), or is called manually with cmake (maybe not using conan at all).

The  naming is bad, the use case is not that clear. Mostly legacy, though cannot be removed or changed, might be actively used.

**Comment by ovidiub13** [comment]
Mark it as deprecated for a few releases, then remove it or rename it.

**Comment by memsharded** [comment]
Not possible until 2.0, which will take a long time.

**Comment by jgsogo** [comment]
@memsharded shall we document the `CONAN_EXPORTED` variable or it's better not to do it so fewer people will be affected by the deprecation?

Maybe we should add now a variable with a proper name, implement and document it?

**Comment by ovidiub13** [comment]
IMO it should be documented for what it does, and renamed possibly in a future release.

**Comment by danimtb** [comment]
closed by conan-io/docs#817
